### PR TITLE
Proposed changes to content templates and guidelines

### PR DESCRIPTION
**Pull request by eddie-knight**

### CODE REVIEW

File: docs/governance/community-guidelines/content-standards-and-practices/control-definitions.md [hunk 1]
```diff
@@ -1,39 +1,56 @@
...
 
-## Control Definition Format
+## Common vs. Specific Controls
 
-In order to create a cohesive standard that is readily useful to end users, controls must be indistinguishable from each other in format, style, and tone. A review from the [Communications WG] is recommended, but not required, in cases where additional support is needed to match the writing style and tone.
+To streamline maintenance, the CCC project maintains a list of [common controls].
 
-Similarly, all controls must match the layout presented in the [control template](../templates/controls.yaml) prior to release.
+Each service category’s `controls.yaml` file references common controls by listing their IDs under the top-level `common_controls` value. During the release pipeline, our [delivery tooling] compiles these common controls into the final document alongside any specific controls. In the final output, both types of controls are presented consistently, with the unique identifier being the only difference.
```
Comment (sshiells-scottlogic) [inline docs/governance/community-guidelines/content-standards-and-practices/control-definitions.md:11]: Broken link

File: docs/governance/community-guidelines/content-standards-and-practices/control-definitions.md [hunk 2]
```diff
@@ -1,39 +1,56 @@
...
+### Control Definition Values
 
-The following list should be updated in the event that a new control family is added to the CCC Controls Catalog:
+When creating a new control definition, use the following values:
+
+- **Control ID** (`id`): A unique identifier for the control, following the format `<category-id>.C<#>`.
+- **Control Title** (`title`): A brief title (3 to 10 words) that succinctly describes the control.
+- **Objective (`objective`)**: A 1 to 3 sentence description outlining the control’s purpose and what it aims to achieve.
+- **Control Family** (`family`): The name of the [Control Family](#control-family) this control belongs to.
+- **CCC Threats** (`threats`): A YAML list of IDs for CCC [threats] that this control is designed to mitigate.
```
Comment (sshiells-scottlogic) [inline docs/governance/community-guidelines/content-standards-and-practices/control-definitions.md:43]: Broken link

File: docs/governance/community-guidelines/content-standards-and-practices/feature-definitions.md [hunk 3]
```diff
@@ -4,23 +4,46 @@ A feature definitions document provides a detailed specification of features ass
 
 Each feature definition should be created for a service in the CCC Taxonomy, with each feature mapped to a specific aspect or functionality of that service.
 
+## Common vs. Specific Features
+
+To streamline maintenance, the CCC project maintains a list of [common features].
+
+Each service category’s `features.yaml` file references common features by listing their IDs under the top-level `common_features` value. During the release pipeline, our [delivery tooling] compiles these common features into the final document alongside any specific features. In the final output, both types of features are presented consistently, with the unique identifier being the only difference.
```
Comment (sshiells-scottlogic) [inline docs/governance/community-guidelines/content-standards-and-practices/feature-definitions.md:11]: Broken link

File: docs/governance/community-guidelines/content-standards-and-practices/threat-definitions.md [hunk 4]
```diff
@@ -1,25 +1,51 @@
...
+- Specific threats are unique to a particular service category.
+- If a threat is relevant to multiple categories, consider whether it should be generalized and added to the common threats list.
+
+## Threat Documentation Process
+
+When creating or updating a `threats.yaml` file for a service category, follow these steps:
+
+1. **Review Common Threats**: Start by reviewing the [common threats] list. If any common threats apply to this category, reference them by adding their IDs to the `common_threats` list.
+2. **Define Specific Threats**: If a threat is unique to the service category, document it in the `threats` section of the `threats.yaml` file.
+3. **Consider Generalization**: If a specific threat could apply to at least three other service categories, evaluate whether it can be generalized and added to the [common threats] list.
```
Comment (sshiells-scottlogic) [inline docs/governance/community-guidelines/content-standards-and-practices/threat-definitions.md:29]: Broken link

File: docs/governance/community-guidelines/content-standards-and-practices/threat-definitions.md [hunk 5]
```diff
@@ -1,25 +1,51 @@
...
 
 ## Threat Definition Format
 
-In order to create a cohesive standard that is readily useful to end users, threats must be indistinguishable from each other in format, style, and tone. A review from the [Communications WG] is recommended, but not required, in cases where additional support is needed to match the writing style and tone.
+To maintain consistency, all threats—whether common or specific—must follow the same format, style, and tone. Each threat should adhere to the [threats template] before release.
```
Comment (sshiells-scottlogic) [inline docs/governance/community-guidelines/content-standards-and-practices/threat-definitions.md:33]: Broken link

**Review by sshiells-scottlogic — COMMENTED** [review]
This looks good to me. I have flagged a few broken links, nothing major.

File: docs/governance/community-guidelines/content-standards-and-practices/control-definitions.md (hunk 1 above)
Comment (eddie-knight) [reply docs/governance/community-guidelines/content-standards-and-practices/control-definitions.md:11]: frick 😆 at least I'm consistent

**Review by sshiells-scottlogic — APPROVED** [review]

**Pull request merged**